### PR TITLE
Portofolio contact form

### PR DESCRIPTION
**Pull request by hopeforkoami**
in this branch, I added the footer and the contact form according to Figma template 3.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,23 +212,64 @@ <h4 class="item_name">Frameworks</h4>
...
-      <div class="contact_text">
+    <footer class="portofolio_footer">
+      <div class="contact_me_section">
+        <h1 class="contact_form_title">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form class="contact_form" action="https://formspree.io/f/mknegjnj" method="POST">
+          <input type="text" class="form_input" placeholder="Full Name" maxlength="30" required>
+          <input type="email" class="form_input" placeholder="Email address" required>
+          <textarea id="contact_text_area" class="form_input" placeholder="write your message here" maxlength="500"
```
Comment (codepantha) [inline index.html:235]: - [ ] An error occurs while trying to submit your form. FormSpree expects you to have the `name` attribute on all your form input elements. This is how FormSpree gets the data in your input. 
For instance:
```
<input type="email" name="email" class="form_input" placeholder="Email address" required>
```

File: index.html [hunk 2]
```diff
@@ -208,23 +212,64 @@ <h4 class="item_name">Frameworks</h4>
...
+            required></textarea>
+          <button class="submit_button" type="submit">Get in touch</button>
+        </form>
```
Comment (codepantha) [inline index.html:238]: - [ ] Please style your button interations such as hover, active and enabled according to the Figma template you have chosen.
### Figma Design
image
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ Used GitHub Flow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] SInce this is a collaborative project, kindly include your coding partners' names in the "Authors" section of your README 🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (hopeforkoami) [reply index.html:238]: thank you

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @hopeforkoami  , 

Great work on making the changes requested by a previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

- [Optional ] Kindly, try to fix the the MIT LICENSE In README file, The Link is broken.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**